src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.trim()) {
+    return {};
+  }
+
+  return sourceString
+    .split(';')
+    .map((keyValue) => keyValue.split(':').map((str) => str.trim()))
```

Comment:
When using 'map', the callback function should be concise. Here, it's used correctly to split the key and value and trim them. However, consider validating the length of the resulting array to ensure there are both a key and a value after the split.